parse-osmium.cpp

```diff
@@ -21,7 +21,10 @@
 */
 
 #include <boost/format.hpp>
-
+#ifdef _WIN32
+#    define NOMINMAX
+#    include <WinSock2.h>
```

Comment:
This was my mistake. A consistent, solution wide use of `WIN32_LEAN_AND_MEAN` takes care of this. The extra include isn't necessary.